packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2223,8 +2270,52 @@ class Row extends Flex {
 /// If you only have one child, then consider using [Align] or [Center] to
 /// position the child.
 ///
+/// ## Sample code
+///
+/// This example uses a [Column] to arrange three widgets vertically, the last
+/// being made to fill all the remaining space.
+///
+/// ```dart
+/// new Column(
+///   children: <Widget>[
+///     new Text('Deliver features faster'),
+///     new Text('Craft beautiful UIs'),
+///     new Expanded(
+///       child: new FittedBox(
+///         fit: BoxFit.contain,
+///         child: const FlutterLogo(),
+///       ),
+///     ),
+///   ],
+/// )
+/// ```
+///
+/// In the sample above, the text and the logo are centered on each line. In the
+/// following example, the [crossAxisAlignment] is set to
+/// [CrossAxisAlignment.start], so that the children are left-aligned. The
```

Comment:
I wonder if terms like 'left-aligned' are going to haunt us when RTL layouts are supported.